covid_xprize/examples/prescriptors/neat/utils.py

```diff
@@ -4,17 +4,29 @@
 import subprocess
 import urllib.request
 import pandas as pd
+from pathlib import Path
+
+from tempfile import NamedTemporaryFile
 
 from covid_xprize.validation.scenario_generator import get_raw_data, generate_scenario
 
+# URL for Oxford data
 DATA_URL = "https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv"
-ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
-DATA_PATH = os.path.join(ROOT_DIR, 'data')
-HIST_DATA_FILE_PATH = os.path.join(DATA_PATH, 'OxCGRT_latest.csv')
 
-PREDICT_MODULE = 'covid_xprize/standard_predictor/predict.py'
-TMP_PRED_FILE_NAME = 'tmp_predictions_for_prescriptions/preds.csv'
-TMP_PRESCRIPTION_FILE = 'tmp_prescription.csv'
+# Path to where this script lives
+ROOT_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
+
+# Data directory (we will download the Oxford data to here)
+DATA_PATH = ROOT_DIR / 'data'
```

Comment:
Use `pathlib` to locate all files relative to where this script lives.

Note the `/` operator is doing more than it appears -- it is overloaded (à la C++) by `pathlib` to do the right thing in synthesizing paths in the correct manner for the local operating system. In _theory_ this "just works" on Windows too but I have no way to test it.